en/releases/1.12.md

```diff
@@ -1,30 +1,69 @@
   * Removes unexpected tilt changes upon reaching travel speed velocity
   * Intuitive shunting e.g. when landing
-  * Comparable to Altitude mode but with the familiar benefits
-    * drift less hover
-    * straight fast forward flight
-    * automatic braking when letting go of the stick
   * Opt out possible using [MPC_POS_MODE](../advanced_config/parameter_reference.md#MPC_POS_MODE)
   * Development: [First attempt](https://github.com/PX4/PX4-Autopilot/pull/12072), [Introduction](https://github.com/PX4/PX4-Autopilot/pull/16052), [Improvements](https://github.com/PX4/PX4-Autopilot/pull/16320), [Bugfix zero oscillation](https://github.com/PX4/PX4-Autopilot/pull/16786), [Bugfix position unlock](https://github.com/PX4/PX4-Autopilot/pull/16791), [Bugfix invalid setpoint](https://github.com/PX4/PX4-Autopilot/pull/17078)
 
-* #### Hover thrust independent velocity control gains
-  * Parameters `MPC_{XY/Z}_VEL_{P/I/D}` were replaced with `MPC_{XY/Z}_VEL_{P/I/D}_ACC`, see:<br/>
+* **Hover thrust independent velocity control gains**
+  * Parameters `MPC_{XY/Z}_VEL_{P/I/D}` were replaced with `MPC_{XY/Z}_VEL_{P/I/D}_ACC`, see: 
    [MPC_XY_VEL_P_ACC](../advanced_config/parameter_reference.md#MPC_XY_VEL_P_ACC), [MPC_XY_VEL_I_ACC](../advanced_config/parameter_reference.md#MPC_XY_VEL_I_ACC), [MPC_XY_VEL_D_ACC](../advanced_config/parameter_reference.md#MPC_XY_VEL_D_ACC), [MPC_Z_VEL_P_ACC](../advanced_config/parameter_reference.md#MPC_Z_VEL_P_ACC), [MPC_Z_VEL_I_ACC](../advanced_config/parameter_reference.md#MPC_Z_VEL_I_ACC), [MPC_Z_VEL_D_ACC](../advanced_config/parameter_reference.md#MPC_Z_VEL_D_ACC)
-  * **Attention:** The gains have a new meaning
-    * Scale from velocity error in m/s to acceleration output in m/s^2
-    * Existing gains need to roughly be rescaled by a factor of `* gravitational constant / hover thrust`
-    * Automatic parameter transition assumes 50% hover thrust: ~10m/s^2 / 50% = 20 m/s^2. See [question](https://github.com/PX4/PX4-Autopilot/pull/14823#issuecomment-791357646)
+   
+    :::warning The gains have a new meaning
+    * Scale from velocity error in $m/s$ to acceleration output in $m/s^2$
+    * Existing gains need to roughly be rescaled by a factor of: $gravitational \_ constant / hover\_thrust$
+    * Automatic parameter transition assumes 50% hover thrust: `~10m/s^2 / 50% = 20 m/s^2`.
+      See [question](https://github.com/PX4/PX4-Autopilot/pull/14823#issuecomment-791357646)
+    :::
   * Development: [Logic introduction](https://github.com/PX4/PX4-Autopilot/pull/14749), [Parameter replacement](https://github.com/PX4/PX4-Autopilot/pull/14823)
+* **Improve Rounded Turns ([PR#16376](https://github.com/PX4/PX4-Autopilot/pull/16376))**
+  * Creates a more rounded turn at waypoints in multirotor missions (using L1-style guidance logic in corners)
+
+### VTOL
+
+* **RTL improvements ([PR#16377](https://github.com/PX4/PX4-Autopilot/pull/16377))**
```

Comment:
@mrpollo this needs docs too - i have started discussion here: https://github.com/PX4/PX4-Autopilot/pull/16377#issuecomment-805537699

Reply:
No answer. Not surprised. RTL behaviour is super complicated and hard to document. This looks like an edge case fix though, so we might be missing something, but what is there probably isn't wrong.